build.sbt

```diff
@@ -204,7 +204,11 @@ lazy val trees = crossProject(allPlatforms: _*).in(file("scalameta/trees")).sett
   // scalacOptions += "-Xprint:typer",
   enableHardcoreMacros,
   libraryDependencies ++= {
-    val fastparseVersion = if (isScala211.value) "3.0.2" else "3.1.0"
+    val fastparseVersion =
+      if (isScala211.value) "3.0.2"
+      else if (VersionNumber(scalaVersion.value).matchesSemVer(SemanticSelector(">=2.13.14")))
```

Comment:
should we invert the check here, so that the most recent version, and the one that will be updated in the future, is the last one?

Reply:
Done